tests/fast/RESTSimKmsVaultUnit.toml

```diff
@@ -0,0 +1,8 @@
+[[test]]
+testTitle = 'RESTSimKmsVaultTest'
```

Comment:
We should avoid adding explicit unit test files unless the unit test can't be run alongside other unit tests for some reason.

Reply:
Sure, `RESTUtilsUnit` can be used to compile unit tests.